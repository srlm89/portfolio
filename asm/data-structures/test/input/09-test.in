ERASE actual/09-test.out
INSERT 8 3.545609 1408.2007.720p.BluRay.x264-SiNNERS
PRINT_IN_ORDER actual/09-test.out
INSERT 9 6.559621 Kingdom of Heaven [2005] [720p]
INSERT 5 6.572835 Core   720p
REMOVE_ID 5
PRINT_IN_ORDER actual/09-test.out
INSERT 7 1.368424 Rocky 2 [1979] [BluRay] [720p] [AC3]
INSERT 10 4.556199 This Is It [2009] [720p]
INSERT 13 0.247650 Orphan (2009)    1080p
REMOVE_ID 7
PRINT_IN_ORDER actual/09-test.out
INSERT 11 5.307836 Disaster Movie 2008   720p
INSERT 14 9.359769 Burn E [720p]
INSERT 1 8.419049 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM
INSERT 15 7.459478 Push 720P
REMOVE_ID 15
REMOVE_ID 8
PRINT_IN_ORDER actual/09-test.out
INSERT 2 8.960071 Terminator Salvation Director's cut  ( 2009)    1080p
INSERT 19 3.612042 Apocalypto   720p
INSERT 0 5.096969 Die Hard 4   1080p
INSERT 21 3.314330 Street Kings   720p
INSERT 18 0.752585 The Mummy Tomb Of The Dragon Emperor   720p
REMOVE_ID 19
REMOVE_ID 11
PRINT_IN_ORDER actual/09-test.out
