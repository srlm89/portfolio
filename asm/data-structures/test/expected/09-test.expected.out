[ 8 3.545609 1408.2007.720p.BluRay.x264-SiNNERS ]
[ 8 3.545609 1408.2007.720p.BluRay.x264-SiNNERS ][ 9 6.559621 Kingdom of Heaven [2005] [720p] ]
[ 8 3.545609 1408.2007.720p.BluRay.x264-SiNNERS ][ 9 6.559621 Kingdom of Heaven [2005] [720p] ][ 10 4.556199 This Is It [2009] [720p] ][ 13 0.247650 Orphan (2009)    1080p ]
[ 1 8.419049 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 9 6.559621 Kingdom of Heaven [2005] [720p] ][ 10 4.556199 This Is It [2009] [720p] ][ 11 5.307836 Disaster Movie 2008   720p ][ 13 0.247650 Orphan (2009)    1080p ][ 14 9.359769 Burn E [720p] ]
[ 0 5.096969 Die Hard 4   1080p ][ 1 8.419049 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 2 8.960071 Terminator Salvation Director's cut  ( 2009)    1080p ][ 9 6.559621 Kingdom of Heaven [2005] [720p] ][ 10 4.556199 This Is It [2009] [720p] ][ 13 0.247650 Orphan (2009)    1080p ][ 14 9.359769 Burn E [720p] ][ 18 0.752585 The Mummy Tomb Of The Dragon Emperor   720p ][ 21 3.314330 Street Kings   720p ]
